app/(onboard)/layout.tsx

```diff
@@ -19,6 +20,7 @@ export default function Layout({ children }: PropsWithChildren) {
             />
           </Link>
         </div>
+        <Toaster />
```

Comment:
This means we have two of this component in separate layouts. Because there's already a `Toaster` in the dashboard layout, you should instead add this to the root layout, and remove it from here and the dashboard layout.

Reply:
resolved with [ea02035](https://github.com/complexdatacollective/Fresco/pull/27/commits/ea020353c797666bf77a7abd67e6ff446c452ac9)